compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/declarations/FunctionParamsTest.java

```diff
@@ -66,4 +66,14 @@ public void testIncompleteBinarExprInDefaultableParams() {
     public void testMoreParamsAfterRestParam() {
         test("func-definition/func_params_source_07.bal", "func-definition/func_params_assert_07.json");
     }
+
+    @Test
+    public void testIncludedRecordParam() {
+        test("func-definition/func_params_source_09.bal", "func-definition/func_params_assert_09.json");
+    }
+
+    @Test
+    public void testIncludedRecordParamWithDefaultValues() {
+        test("func-definition/func_params_source_10.bal", "func-definition/func_params_assert_10.json");
+    }
```

Comment:
Shall we add some negative syntax tests as well?
e.g: additional tokens before and after asterisk, missing tokens before and after asterisk, etc

Reply:
Added.